src/Core/Prefab/GlobeView.js

```diff
@@ -229,11 +230,14 @@ function GlobeView(viewerDiv, coordCarto, options) {
     this._renderState = RendererConstant.FINAL;
 
     this.preRender = () => {
-        var len = this.camera.position().length();
-        var lim = size * 1.1;
+        const v = new THREE.Vector3();
+        v.setFromMatrixPosition(wgs84TileLayer.object.matrixWorld);
+        var len = v.distanceTo(this.camera.position());
+        v.setFromMatrixScale(wgs84TileLayer.object.matrixWorld);
+        var lim = v.x * size * 1.1;
 
         if (len < lim) {
-            var t = Math.pow(Math.cos((lim - len) / (lim - size * 0.9981) * Math.PI * 0.5), 1.5);
+            var t = Math.pow(Math.cos((lim - len) / (lim - v.x * size * 0.9981) * Math.PI * 0.5), 1.5);
```

Comment:
`v` is a temp vector, and at this point it holds the `scale` of the object owning the globe.
So we scale the ellipsoid size approximation (`size`) when using it to compute `t`.